CHANGES.md

```diff
@@ -24,6 +24,56 @@ this file to understand what changed.
 - bug fixes
 - other
 
+## v0.29.0
+
+- educational materials
+- Doc update for `responsibleai_vision` by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2164
+- [Object Detection] User Flows / Architectures by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2134
+- [Object Detection] DFD Documentation by @natalie-isak in https://github.com/microsoft/responsible-ai-toolbox/pull/2138
+- [Object Detection] Explanation Documentation by @natalie-isak in https://github.com/microsoft/responsible-ai-toolbox/pull/2136
+- [Object Detection] CanvasTools documentation by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2133
+- [Object Detection] Model Overview Object Detection Documentation by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2132
+- new features
+  - ## Responsible AI Dashboard
+  - [Object Detection] Cache support for Model Overview Metrics by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2158
+  - [Image Explorer] CanvasTools Image Loading support for Object Detection by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2097
+  - [Object Detection] [Image Explorer] Bounding Box Support with CanvasTools on Flyout by @Advitya17 in https://github.com/microsoft/responsible-ai-toolbox/pull/2116
+- bug fixes and tests
+  - ## Responsible AI Dashboard
+  - update responsibleai-text and responsibleai-vision packages to latest responsibleai 0.28.0 release by @imatiach-msft in https://github.com/microsoft/responsible-ai-toolbox/pull/2159
+  - Robot page exception in DatasetUtils where jointDataset.metaDict[property] is undefined by @tongyu-microsoft in https://github.com/microsoft/responsible-ai-toolbox/pull/2148
+  - Fix individual causal plot not rendering for 5k points by @vinuthakaranth in https://github.com/microsoft/responsible-ai-toolbox/pull/2113
+  - Remove console log by @RubyZ10 in https://github.com/microsoft/responsible-ai-toolbox/pull/2125
+  - Refactor- Add datasetCohorts to ModelAssessementContext && create datasetCohort && add compositeFilter in DatasetCohort by @RubyZ10 in https://github.com/microsoft/responsible-ai-toolbox/pull/2119
+  - Add loading spinner for causal policy tab by @gaugup in https://github.com/microsoft/responsible-ai-toolbox/pull/2121
```

Comment:
should these go under a "Causal" section?  We use to have one in previous releases.  Same for line 45 as well?